### PR TITLE
[EEG] new table to track EEG additional events

### PR DESCRIPTION
**Pull request by regisoc**
## Brief summary of changes

This PR adds a new table to keep additional data from BIDS event files (Loris-MRI).
Currently, additional data are ignored.

Linked to:

- [Loris-MRI 835](https://github.com/aces/Loris-MRI/issues/835)
- [Loris-MRI 873](https://github.com/aces/Loris-MRI/pull/873)

### CODE REVIEW

**Review by laemtl — COMMENTED** [review]
We also need to update SQL/0000-00-05-ElectrophysiologyTables.sql and /raisinbread/RB_files. What about the optional fields in  `physiological_task_event`? Should we migrate them in the new table?

File: SQL/New_patches/2022-11-22-eeg-additional-events-table.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+-- Create `physiological_task_event_opt` table
+-- tracks additional events from bids archives
+CREATE TABLE `physiological_task_event_opt` (
+    `ID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+    `PhysiologicalTaskEventID` int(10) unsigned NOT NULL,
+    `TaskName` varchar(50) NOT NULL,
+    `TaskValue` varchar(255) NOT NULL,
```
[the remark this answers is not in the document]
Comment (laemtl) [reply SQL/New_patches/2022-11-22-eeg-additional-events-table.sql:7]: A Null value can have different meanings. The key/value pait can exist, but with a NULL value to indicate that the field does not apply for example.

[the remark this answers is not in the document]
Comment (regisoc) [reply SQL/New_patches/2022-11-22-eeg-additional-events-table.sql:7]: Oh ok, I know what you mean. There are 2 things:
1. If we want to migrate **optional fields** from `physiological_task_event` into `physiological_task_event_opt`, allowing NULL values should definitively be there. But...
2. This PR does not take that path. It allows tracking **additional fields** only by putting them into another table. It does not aim to change the original behavior. If there are no additional fields, there will be no entry for the associated event in this table.

Do we need to migrate the **optional fields**? → Is there any strong incentive to do so?

[the remark this answers is not in the document]
Comment (regisoc) [reply SQL/New_patches/2022-11-22-eeg-additional-events-table.sql]: After discussion, `taskValues` will be nullable

**Comment by regisoc** [comment]
- [x] task value nullable
- [x] insert into schema
- [ ] insert into RB

**Comment by regisoc** [comment]
Rebased/Changed for #8332.

**Review by laemtl — APPROVED** [review]

**Comment by christinerogers** [comment]
@regisoc could you kindly update this PR title and description if it doesn't cover add EEG additional events, only links events to Tasks?   (otherwise we will have seemingly duplicate PRs)

cc @jeffersoncasimir @laemtl

**Comment by laemtl** [comment]
@christinerogers Unfortunately, the PR has been merged so it cannot be edited.

**Comment by regisoc** [comment]
Just updated the description @christinerogers, thanks for your feedback!

**Pull request merged**